lib/polymorpheus/mysql_adapter/foreigner_constraints.rb

```diff
@@ -0,0 +1,29 @@
+module Polymorpheus
+  module ConnectionAdapters
+    module MysqlAdapter
+      def generate_constraints(options)
+        constraints = []
+
+        ['delete', 'update'].each do |event|
+          option = "on_#{event}".to_sym
+          next unless options.has_key?(option)
+
+          action = case options[option].to_sym
```

Comment:
a bit of extra safety in case the value does not respond to `to_sym`, I suggest:
```
action = case (options[option].to_sym rescue nil)
```

OR

on line 9 above
```
next unless options.has_key?(option) and options[option].respond_to?(:to_sym)
```